DB/src/main/java/io/deephaven/db/tables/TableDefinition.java

```diff
@@ -705,10 +602,6 @@ public static TableDefinition createUserPartitionedTableDefinition(@NotNull fina
         return result;
     }
 
-    // TODO: DELETE THESE OVERRIDES AND RELATED CODE IN persist.*() METHODS (SEE NOTES BELOW)
```

Comment:
You appear to have deleted the TODO without deleting the offending code. Maybe save it for the ADO-removal PR?